src/test/ebpf_yaml.cpp

```
@@ -358,64 +359,52 @@ ConformanceTestResult run_conformance_test_case(const std::vector<std::byte>& me
+        const cfg_t cfg = prepare_cfg(prog, info, options.cfg_opts);
+        auto invariants = analyze(cfg, pre_invariant);
+        assert(invariants);
+        return ConformanceTestResult{.success = invariants->verified(cfg), .r0_value = invariants->exit_value()};
     } catch (const std::exception&) {
         // Catch exceptions thrown in ebpf_domain.cpp.
         return {};
     }
 }
 
-void print_failure(const Failure& failure, std::ostream& out) {
+void print_failure(const Failure& failure, std::ostream& os) {
```

cppcheck

[style] The function 'print_failure' is never used.

(unusedFunction)